tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -6,67 +6,74 @@
+#
 #
 # License: BSD-3-Clause
 
 # %%
 # Data loading
 # ------------
 #
-# First we will load an eye tracker recording from SR research's proprietary
-# ``'.asc'`` file format.
+# As usual we start by importing the modules we need and loading some
+# :ref:`example data <eyelink-dataset>`: eye-tracking data recorded from SR research's
+# ``'.asc'`` file format, and EEG data recorded from EGI's ``'.mff'`` file format.
 #
-# The info structure tells us we loaded a monocular recording with 2
-# ``'eyegaze'``, channels (X/Y), 1 ``'pupil'`` channel, and 1 ``'stim'``
-# channel.
+# The info structure of the eye-tracking data tells us we loaded a monocular recording
+# with 2 ``'eyegaze'``, channels (X/Y), 1 ``'pupil'`` channel, 1 ``'stim'`` channel, and
+# 3 channels for the head distance and position (since this data was collected using
+# EyeLink's Remote mode).
 
 import mne
 from mne.datasets.eyelink import data_path
 from mne.preprocessing.eyetracking import read_eyelink_calibration
 
-eyelink_fname = data_path() / "mono_multi-block_multi-DINS.asc"
+et_fpath = data_path() / "sub-01_task-plr_eyetrack.asc"
+eeg_fpath = data_path() / "sub-01_task-plr_eeg.mff"
+
+raw_et = mne.io.read_raw_eyelink(et_fpath, create_annotations=["blinks"])
+raw_eeg = mne.io.read_raw_egi(eeg_fpath, preload=True, verbose="warning")
+raw_eeg.filter(1, 30, verbose="warning")
+print(f"RawEyelink info: {raw_et.info}")
 
-raw = mne.io.read_raw_eyelink(eyelink_fname, create_annotations=["blinks", "messages"])
-raw.crop(tmin=0, tmax=130)  # for this demonstration, let's take a subset of the data
+# %%
+# .. seealso:: :ref:`tut-importing-eyetracking-data`
 
 # %%
 # Ocular annotations
 # ------------------
-# By default, Eyelink files will output events for ocular events (blinks,
+# By default, EyeLink files will output events for ocular events (blinks,
 # saccades, fixations), and experiment messages. MNE will store these events
 # as `mne.Annotations`. Ocular annotations contain channel information, in the
-# ``'ch_names'``` key. This means that we can see which eye an ocular event occurred in:
+# ``'ch_names'`` key. This means that we can see which eye an ocular event occurred in,
+# which can be useful for binocular recordings:
 
-print(raw.annotations[0])  # a blink in the right eye
+print(raw_et.annotations[0])  # a blink in the right eye
 
 # %%
 # If we are only interested in certain event types from
-# the Eyelink file, we can select for these using the ``'create_annotations'``
-# argument of `mne.io.read_raw_eyelink`. above, we only created annotations
-# for blinks, and experiment messages.
-#
-# Note that ``'blink'`` annotations are read in as ``'BAD_blink'``, and MNE will treat
-# these as bad segments of data. This means that blink periods will be dropped during
-# epoching by default.
+# the EyeLink file, we can select for these using the ``'create_annotations'``
+# argument of `~mne.io.read_raw_eyelink`. above, we only created annotations
```

Comment:
```suggestion
# argument of `~mne.io.read_raw_eyelink`. Above, we only created annotations
```
Slightly out of scope here, but we might as well fix this typo?

Reply:
ah yes thank you, I keep missing capitalizations!